Execution failed for task ':app:validateSigningDebug'.

**pg-sdk-react-native-sample\android\app\debug.keystore' not found for signing config 'debug'.**


npx react-native run-android
info Starting JS server...
info Installing the app...

> Task :react-native-phonepe-pg:compileDebugKotlin
'compileDebugJavaWithJavac' task (current target is 11) and 'compileDebugKotlin' task (current target is 1.8) jvm target compatibility should be set to the same Java version.
w: C:\Users\MSS Dev\Desktop\pg-sdk-react-native-sample\node_modules\react-native-phonepe-pg\android\src\main\java\com\phonepepaymentsdk\DataUtil.kt: (26, 
67): 'get(String!): Any?' is deprecated. Deprecated in Java
w: C:\Users\MSS Dev\Desktop\pg-sdk-react-native-sample\node_modules\react-native-phonepe-pg\android\src\main\java\com\phonepepaymentsdk\PhonePePaymentSDKModule.kt: (143, 9): Parameter 'headers' is never used
w: C:\Users\MSS Dev\Desktop\pg-sdk-react-native-sample\node_modules\react-native-phonepe-pg\android\src\main\java\com\phonepepaymentsdk\PhonePePaymentSDKModule.kt: (145, 9): Parameter 'callBackURL' is never used
w: C:\Users\MSS Dev\Desktop\pg-sdk-react-native-sample\node_modules\react-native-phonepe-pg\android\src\main\java\com\phonepepaymentsdk\PhonePePaymentSDKModule.kt: (178, 9): Parameter 'headers' is never used
w: C:\Users\MSS Dev\Desktop\pg-sdk-react-native-sample\node_modules\react-native-phonepe-pg\android\src\main\java\com\phonepepaymentsdk\PhonePePaymentSDKModule.kt: (179, 9): Parameter 'callBackURL' is never used

> Task :app:validateSigningDebug FAILED
50 actionable tasks: 45 executed, 5 up-to-date

info 💡 Tip: Make sure that you have set up your development environment correctly, by running react-native doctor. To read more about doctor command visit: https://github.com/react-native-community/cli/blob/main/packages/cli-doctor/README.md#doctor


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:validateSigningDebug'.
> Keystore file 'C:\Users\MSS Dev\Desktop\pg-sdk-react-native-sample\android\app\debug.keystore' not found for signing config 'debug'.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 9m 23s
error Failed to install the app.
info Run CLI with --verbose flag for more details.

--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -81,22 +81,11 @@ android {
         versionCode 1
         versionName "1.0"
     }
-    signingConfigs {
-        debug {
-            storeFile file('debug.keystore')
-            storePassword 'android'
-            keyAlias 'androiddebugkey'
-            keyPassword 'android'
-        }
-    }
+
     buildTypes {
         debug {
-            signingConfig signingConfigs.debug
         }
         release {
-            // Caution! In production, you need to generate your own keystore file.
-            // see https://reactnative.dev/docs/signed-apk-android.
-            signingConfig signingConfigs.debug
             minifyEnabled enableProguardInReleaseBuilds
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }

